modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/MetadataUpdater_conf.pm

```diff
@@ -0,0 +1,206 @@
+=pod 
+=head1 NAME
+
+=head1 SYNOPSIS
+
+=head1 DESCRIPTION  
+
+=head1 LICENSE
+    Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+    Copyright [2016-2022] EMBL-European Bioinformatics Institute
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+         http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and limitations under the License.
+=head1 CONTACT
+    Please subscribe to the Hive mailing list:  http://listserver.ebi.ac.uk/mailman/listinfo/ehive-users  to discuss Hive-related questions or to be notified of our updates
+=cut
+
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::MetadataUpdater_conf;
+
+use strict;
+use warnings;
+use Data::Dumper;
+
+use base ('Bio::EnsEMBL::Production::Pipeline::PipeConfig::BasePython_conf');
+
+sub default_options {
+    my ($self) = @_;
+    return {
+        %{$self->SUPER::default_options},
+        pipeline_name => 'metadata_updater',
+        metadata_uri  => '',
```

Comment:
With empty params default, it will be possible to kick off without them if they are mandatory, they should be `undef` I guess.